int64 xDistance # distance between x coordinates of the stationary and moving turtles
int64 yDistance # distance between y coordinates of the stationary and moving turtles
int64 distance # distance betweent the two turtles 
